2005-11-07  Kazuhiro Inaoka  <inaoka.kazuhiro@renesas.com>

        * elf/elf.h (R_M32R_REL32): New macro.

Index: elf/elf.h
===================================================================
RCS file: /cvs/glibc/libc/elf/elf.h,v
retrieving revision 1.143
diff -u -p -r1.143 elf.h
--- elf/elf.h	12 Nov 2004 21:51:33 -0000	1.143
+++ elf/elf.h	9 May 2006 05:46:38 -0000
@@ -2525,6 +2525,7 @@ typedef Elf32_Addr Elf32_Conflict;
 #define R_M32R_SDA16_RELA	42	/* 16 bit offset in SDA */
 #define R_M32R_RELA_GNU_VTINHERIT	43
 #define R_M32R_RELA_GNU_VTENTRY	44
+#define R_M32R_REL32		45	/* PC relative 32 bit */
 
 #define R_M32R_GOT24		48	/* 24 bit GOT entry */
 #define R_M32R_26_PLTREL	49	/* 26 bit PC relative to PLT shifted */
